### ISSUE TITLE
[FEATURE REQUEST] Support for RC-Servo operated valves 

### ISSUE THREAD

**Issue by stefan63** [DeeEmm/DIY-Flow-Bench#219]
I realize that stepper motor is in the todo list, but I stll want to lift the posibilty to use RC-servos to controll valves.

**Comment by DeeEmm** [comment]
Is this PWM output?

**Comment by stefan63** [comment]
Yes and there are loads of code hints for esp32 as well as Arduino in general.
https://en.wikipedia.org/wiki/Servo_control

**Comment by DeeEmm** [comment]
PWM is pretty trivial to implement but it will require additions electronics as I suspect that the servomotor will exceed the capabilities of standard output current capacity (nominal 500mA). This is easy enough to do using a mosFET. The new PCB also now has a prototype area where additional electronics like this can be added which includes direct access to molex pins.

I suspect that we will also use the same philosophy for PWM based IAT bypass valve to fine control the bleed pressure

**Comment by KirikauKiwi** [comment]
The PWM implementation will also be useful for motor control as discussed in #186 Good for DIYFB V3
